src/bin/pg_autoctl/main.c

```diff
@@ -48,6 +51,16 @@ main(int argc, char **argv)
 	/* set our logging infrastructure */
 	(void) set_logger();
 
+	/*
+	 * Since PG 12, we need to call pg_logging_init before any calls to pg_log_*
+	 * otherwise, we get a segfault. Although we doesn't use pg_log_* directly,
+	 * functions from the common library such as rmtree do use them.
+	 * Logging change introduced in PG 12: https://git.io/JqvcN
```

Comment:
Is the short URL scheme going to resist the test of time? I think we should use either https://git.postgresql.org/gitweb/?p=postgresql.git;a=commitdiff;h=cc8d41511721d25d557fc02a46c053c0a602fed0 or https://git.postgresql.org/cgit/postgresql.git/commit/?id=cc8d41511721d25d557fc02a46c053c0a602fed0.

Reply:
git.io is maintained by Github itself so thought it might be fine, but who knows, valid concern.
Changed it to https://git.postgresql.org/cgit/postgresql.git/commit/?id=cc8d41511721d25d557fc02a46c053c0a602fed0